client/Component/styles/gallery.css

```diff
@@ -1,41 +1,67 @@
 .container {
-    width: 100%;
-    height: 400.000px;
-    background-color: gray;
-    display: grid;
-    grid-column-gap: 1px;
-    grid-row-gap: 1px;
-    object-fit: contain;
+  width: 100%;
+  height: 400.000px;
+  background-color: black;
+  display: grid;
+  grid-column-gap: 2px;
+  grid-row-gap: 2px;
+  object-fit: contain;
+  overflow: hidden;
+}
+
+.divInner {
+  height: 100%;
+  width: 100%;
+  background-size: cover;
+  transition: transform .4s;
+  cursor: pointer;
+}
+.divInnerDarken {
+  height: 100%;
+  width: 100%;
+  background-size: cover;
+  transition: transform 1s;
+  filter: brightness(75%); 
+  cursor: pointer;
+
+}
+.divInner:hover {
+  transform: scale(1.1);
```

Comment:
The point of using a css module is to avoid conflicts in DOM elements class and id names. It does so by generating a random hash at the end of the class name. 
You would need to configure the module with webpack in such a way that it does that. I am not too clear on how that is implemented. 
I switched to styled components and I think it's much easier to use. If you wish to stick with css module, make sure that a random hash is generated upon rendering of the page.